src/Apps/CollectorProfile/Routes/Saves/Components/SelectArtworkListsModal/SelectArtworkListsModal.tsx

```diff
@@ -226,7 +226,7 @@ export const SelectArtworkListsModalFragmentContainer = createFragmentContainer(
           first: 30
           default: false
           saves: true
-          sort: CREATED_AT_DESC
+          sort: UPDATED_AT_DESC
```

Comment:
I think this change is not related to the ticket (maybe ended up here accidentally @olerichter00?)